cfme/tests/automate/test_instance.py

```diff
@@ -354,3 +358,130 @@ def test_quota_source_value(request, entity, search, copy_quota_instance, generi
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(lambda: provision_request.remove_request(method="rest"))
     assert result.validate(wait="60s")
+
+
+@pytest.fixture(scope='module')
+def setup(domain, klass, namespace):
+    """This fixture creates common domain, namespace, two classes, instance and method. This setup
+    is common for both parameterized tests"""
+
+    klass.schema.add_fields(*[{'name': state, 'type': 'State'}
+                              for state in [STATE1, STATE2, STATE3]])
```

Comment:
I.e. Same comment as above, can you do `for state in fauxfactory.gen_alpha()`?

Reply:
Oh I see, you are using them below, but is it possible to define them in the fixture? And have them be yielded. I will leave this as _optional_